src/lib/devtools/formatters.cljs

```diff
@@ -14,7 +14,8 @@
 
 (defn ^:dynamic available? []
   (or (in-node-context?)                                                                                                      ; node.js or Chrome 47+
-      (and (isChrome) (isVersionOrHigher 47))))
+      (and (isAtLeast "CHROMIUM" 47))
+	  (and (isAtLeast "FIREFOX" 110))))
```

Comment:
```
(defn ^:dynamic available? []
  (or (in-node-context?)
      (isAtLeast "CHROMIUM" 47)
      (isAtLeast "EDGE" 79) ;; First Edge to use Blink, has Blink 79.
      (isAtLeast "FIREFOX" 110)))
```

Reply:
Done that and added Edge to the documentation. I also updated the Firefox version to 111 because the final parsing of custom formatters didn't land yet.